src/Exceptions/InteropBindingException.cs

```diff
@@ -1,20 +1,23 @@
 using System.Text;
+using Concordium.Sdk.Interop;
 
 namespace Concordium.Sdk.Exceptions;
 
 /// <summary>
 /// Thrown when a interop call failed with possible error as message.
 /// </summary>
-internal sealed class InteropBindingException : Exception
+public sealed class InteropBindingException : Exception
```

Comment:
**Discussion:** If we are making this public, we should probably rename it, since the current name is more related to implementation details.
Maybe `SchemaJsonException`?